modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/MultiSelectTag.java

```diff
@@ -0,0 +1,232 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.frontend.taglib.clay.servlet.taglib;
+
+import com.liferay.frontend.taglib.clay.internal.servlet.taglib.BaseContainerTag;
+import com.liferay.portal.kernel.util.Validator;
+
+import java.util.ArrayList;
+import java.util.Map;
+import java.util.Set;
+
+import javax.servlet.jsp.JspException;
+import javax.servlet.jsp.JspWriter;
+
+/**
+ * @author Kresimir Coko
+ */
+public class MultiSelectTag extends BaseContainerTag {
+
+	@Override
+	public int doStartTag() throws JspException {
+		setAttributeNamespace(_ATTRIBUTE_NAMESPACE);
+
+		return super.doStartTag();
+	}
+
+	public String getClearAllTitle() {
+		return _clearAllTitle;
+	}
+
+	public String getCloseButtonAriaLabel() {
+		return _closeButtonAriaLabel;
+	}
+
+	public boolean getDisabledClearAll() {
+		return _disabledClearAll;
+	}
+
+	public Object getFilter() {
+		return _filter;
+	}
+
+	public String getInputName() {
+		return _inputName;
+	}
+
+	public String getInputValue() {
+		return _inputValue;
+	}
+
+	public boolean getIsValid() {
+		return _isValid;
+	}
+
+	public ArrayList<Object> getItems() {
```

Comment:
What items, selectedItems? autocompleteItems? Looking at the clay component it seems to be the items that will be shown as label. Those, to me, are `selectedItems`, which btw are already present in the soy version of the Tag.

Reply:
@carloslancha yea, `selectedItems` is present in the soy version, but it's not wired in any special way, so not sure how it matters for this component.

Reply:
One more thing to note, `items` are the only items in the Clay API, and with @jbalsas' comment a couple days ago 
> As such, we could allow ourselves to be a bit more lenient here if backwards compat is going to complicate it.

Should we worry about having `selectedItems` and `autocompleteItems` or the Clay variant: `items` and `sourceItems`? What other props should we completely deprecate?

Reply:
I mean that we already have an attribute called `selectedItems` so we can use it here instead `items`. Just rename `items` to `selectedItems` xD.

> What other props should we completely deprecate?

Just compose an API that makes sense and that is understandable and, after that, we can deprecate the old method that we don't need anymore.